src/content/docs/en/concepts/why-astro.mdx

```diff
@@ -1,45 +1,59 @@
+**Astro was designed for building content-rich websites.** This includes marketing sites, publishing sites, documentation sites, blogs, portfolios, landing pages, community sites, and ecommerce sites. If you have content to show, it needs to reach your reader quickly.
 
-By contrast, most modern web frameworks are designed for building *web applications*. These frameworks work best for building more complex, application-like experiences in the browser: logged-in admin dashboards, inboxes, social networks, todo lists, and even native-like applications like [Figma](https://figma.com/) and [Ping](https://ping.gg/).
+By contrast, most modern web frameworks were designed for building *web applications*. These frameworks excel at building more complex, application-like experiences in the browser: logged-in admin dashboards, inboxes, social networks, todo lists, and even native-like applications like [Figma](https://figma.com/) and [Ping](https://ping.gg/). However, they often struggle to provide great performance when you need it most.
 
-This is one of the most important differences to understand about Astro. Astro's unique focus on content lets Astro make tradeoffs and deliver unmatched performance features that wouldn't make sense for more application-focused web frameworks to implement.
+Astro's focus on content from its beginnings as a static site builder have allowed Astro to **sensibly scale up to performant, powerful, dynamic web applications** that still respect your content and your audience. Astro's unique focus on content lets Astro make tradeoffs and deliver unmatched performance features that wouldn't make sense for more application-focused web frameworks to implement.
 
-:::tip
-If your project falls into the second "application" camp, Astro might not be the right choice for your project... **and that's okay!** Check out [Next.js](https://nextjs.org/) for a more application-focused alternative to Astro.
-:::
+### Server-first
 
+**Astro leverages [server-rendering](/en/core-conceptes/rendering-modes/) over client-side rendering in the browser as much as possible.** This is the same approach that traditional server-side frameworks -- PHP, WordPress, Laravel, Ruby on Rails, etc. -- have been using for decades. But you don't need to learn a second server-side language to unlock it. With Astro, everything is still just HTML, CSS, and JavaScript (or TypeScript, if you prefer).
 
-## Server-first
+This approach stands in contrast to other modern JavaScript web frameworks like Next.js, SvelteKit, Nuxt, Remix, and others. These frameworks require client-side rendering of your entire website and include server-side rendering mainly to address performance concerns. This approach has been dubbed the **Single-Page App (SPA)**, in contrast with Astro's **Multi-Page App (MPA)** approach.
```

Comment:
Astro's emphasis on server-side rendering does contrast with these frameworks, but it might no be accurate to say that they require client-side rendering for the entire website. Frameworks like Next.js, SvelteKit, and Nuxt offer client-side rendering capabilities, they are not limited to SPA approaches. Each framework provides a range of rendering options, including server-side rendering, static site generation, and hybrid approaches. 

```suggestion
This approach differentiates itself from other modern JavaScript web frameworks such as Next.js, SvelteKit, Nuxt, Remix and others. While these frameworks offer client-side rendering capabilities, they are not limited to SPA (Single-Page Application) approaches. For example, Next.js, SvelteKit, and Nuxt support a hybrid rendering model that combines server-side rendering, static site generation, and client-side rendering, depending on the specific needs of each page. This flexibility allows for performance optimization and enhanced SEO. In contrast, Astro primarily focuses on a server-side rendering approach, akin to a Multi-Page Application (MPA) model, which can be more suitable for content-heavy sites where first-load performance is crucial.
```

Reply:
I agree with you things have changed since we originally wrote this statement, but I don't think we need to get into more of a definition of other services here to be both accurate and get our point across.

The intention (which maybe could be better worded!) is that these *are* primarily client-side frameworks who *had to introduce* any server-rendering that they might now have or do for performance concerns. (i.e. it's not about describing what they support now, it's about the different paths the projects have taken)

The intent isn't necessarily to explain them in detail, but to highlight the difference between them being *client-first*, then adding on some server stuff because they found they needed it, vs Astro being *server-first* from the beginning. 

So I'd be happy to reconsider wording that doesn't go into everyone's options, but again, highlights "where each project ultimately came from" in their approaches, because that *does* have an effect on legacy code, design decisions and even what new people using defaults are nudged into etc., even if everyone now offers some kind of hybrid rendering. I'd like for us to get *that* idea across.